templates/esnli/templates.yaml

```diff
@@ -0,0 +1,119 @@
+dataset: esnli
+templates:
+  16206cff-dd01-408e-b0b4-daabb750b38b: !Template
+    id: 16206cff-dd01-408e-b0b4-daabb750b38b
+    jinja: 'Explain why the relation between the following two sentences can be described
+      as {{ ["an entailment", "neutral", "a contradiction"][label] }}.
+
+
+      Sentence 1: {{premise}}
+
+
+      Sentence 2: {{hypothesis}}
+
+      |||
+
+      {{explanation_1}}'
+    name: '  explain_3'
+    reference: ''
+  3d077f02-bba0-4019-bacd-bf3e7119b1f7: !Template
+    id: 3d077f02-bba0-4019-bacd-bf3e7119b1f7
+    jinja: "{% if label == 0 %} \nWhy does the first sentence entail the second?\n\
+      {% elif label == 1 %}\nWhy are the two sentence neutral?\n{% else %}\nWhy do\

```

Comment:
It might be more natural to say something like "Why do the two sentences neither entail nor contradict each other?" In isolation, there isn't a lot of context to infer what neutral means here.